ganttproject/src/main/java/biz/ganttproject/ganttview/TaskTable.kt

```diff
@@ -658,10 +659,26 @@ class TaskTable(
         }
       }
       initializationCompleted()
+      updateFilterComponent(filteredCount)
     }
     LOGGER.debug("Sync <<<<<<<<<<<<<<<<<")
   }
 
+  private fun updateFilterComponent(filteredCount: Int) {
+    if (::tableMenuLabel.isInitialized) {
+      Platform.runLater {
+        if (this.filters.activeFilter != VOID_FILTER) {
+          tableMenuLabel.text = filteredCount.toString() + "/" + taskManager.taskCount
```

Comment:
Please i18n this string. You can use placeholders {0} and {1} in the string, e.g. "tasks shown: {0} of {1}"